opentracing-impl-java8/src/main/java/io/opentracing/AbstractSpan.java

```diff
@@ -23,14 +23,14 @@
 
 abstract class AbstractSpan implements Span, SpanContext {
 
-    final String operationName;
+    protected final String operationName;
 
     protected final Map<String,String> baggage = new HashMap<>();
 
-    private final Instant start;
+    protected final Instant start;
     private Duration duration;
-    private final Map<String,Object> tags = new HashMap<>();
-    private final List<LogData> logs = new ArrayList<>();
+    protected final Map<String,Object> tags = new HashMap<>();
+    protected final List<LogData> logs = new ArrayList<>();
```

Comment:
It would be better imo to expose a protected final getter that returns an immutable/unmodifiable copy.